internal/core/legacy/dao/notify_group.go

```diff
@@ -191,3 +191,23 @@ func (client *DBClient) DeleteNotifyRelationsByScope(scopeType apistructs.ScopeT
 	err = client.Where("scope_type = ? and scope_id = ? ", scopeType, scopeID).Delete(&model.Notify{}).Error
 	return err
 }
+
+type SPMonitor struct {
```

Comment:
It's strange to query spot related tables inside core services.

Reply:
right~, I think we could get `scopeType`,`scopeID` from label field in record